src/main/java/seedu/address/MainApp.java

```diff
@@ -165,10 +171,35 @@ protected UserPrefs initPrefs(UserPrefsStorage storage) {
         return initializedPrefs;
     }
 
+    /**
+     * Checks if the user data is stores.
```

Comment:
Slight grammar error, I believe it should be "Check if the user data is stored."